.env.example

```diff
@@ -4,11 +4,7 @@
 # 3. Set right vaules for VAULT_URL_FOR_DYNACONF, VAULT_MOUNT_POINT_FOR_DYNACONF and VAULT_PATH_FOR_DYNACONF
 # 4. Run 'make vault-login' to login into vault and to generate and set th token automatically
 # 5. To use secret from vault for any settings in conf/*.yaml, use the format: '@format {this._secret_name_in_vault_}'
-# 6. bugzilla.yaml Example:
-#    BUGZILLA:
-#      URL: https://bugzilla.redhat.com
-#      API_KEY: '@format {this.vault_bz-api-key}'
```

Comment:
I'd keep the example. We can switch `bugzilla.yaml` with `jira.yaml`.

Reply:
Updated the PR.